### ISSUE TITLE
Ctrl+Enter returns error when vacuum command is present in the editor

### ISSUE THREAD

**Issue by jurijcz** [dbeaver/dbeaver#29320]
### Description

The documentation says: to execute a query **under the cursor**, press `Ctrl+Enter`.

But when the editor contains these 2 queries:
```sql
select * from tablename;

vacuum full freeze verbose tablename;
```
`Ctrl+Enter` key combination returns error:
`SQL Error [25001]: ERROR: VACUUM cannot be executed within a pipeline`
in both cases — when the active cursor is on one or the other query.

However, if queries are being swapped:
```sql
vacuum full freeze verbose tablename;

select * from tablename;
```
each one executed normally without errors.

But when adding one another:
```sql
vacuum full freeze verbose tablename;

select * from tablename;

vacuum full freeze verbose othertablename;
```
only first query executed without errors.

### DBeaver Version

Community Edition 24.0.3

### Database and driver

PostgreSQL 16.2
PostgreSQL JDBC Driver 42.7.3

**Comment by E1izabeth** [comment]
Duplicate of https://github.com/dbeaver/dbeaver/issues/26416
The fix will be included in DBeaver 24.0.4.
Also you can try Early Access version with fix https://dbeaver.io/files/ea/ now.

**Comment by purushottamnawale** [comment]
The issue is not yet fixed in version 24.2.2.  
```
SQL Error [25001]: ERROR: VACUUM cannot be executed within a pipeline
```
It seems that multiple VACUUM queries cannot be executed at once. However, it does work in PSQL.
